utilities/utility.py

```diff
@@ -29,6 +40,21 @@ def validate_country_field(**kwargs):
         raise AttributeError("Not a valid country")
 
 
+def validate_name_field(**kwargs):
+    """
+    Function to validate location fields when
+    saving an object
+    :params kwargs
```

Comment:
@michael-basweti  You can also consider using `validate_location_name_field` here.